docs/_attic/sdk/core/examples/app1.go

```diff
@@ -154,7 +154,7 @@ func handleFrom(store sdk.KVStore, from sdk.AccAddress, amt sdk.Coins) sdk.Resul
 	senderCoins := acc.Coins.Minus(amt)
 
 	// If any coin has negative amount, return insufficient coins error.
-	if !senderCoins.IsNotNegative() {
+	if !senderCoins.IsAllGTEZero() {
```

Comment:
I think it'll be clearer if we have `coins#IsAllNegative`.

Reply:
I made it into `IsAnyNegative()`